packages/modules/viewer-react/src/types.ts

```diff
@@ -112,6 +113,8 @@ export interface ItwinViewerInitializerParams {
   enablePerformanceMonitors: boolean;
   /** options for TileAdmin initialization */
   tileAdminOptions?: TileAdmin.Props;
+  /** options for RenderSystem initialization */
+  renderSys?: RenderSystem.Options;
```

Comment:
I think this should be of type: `RenderSystem | RenderSystem.Options`

https://www.itwinjs.org/reference/core-frontend/imodelapp/imodelappoptions/